google/cloud/spanner_v1/batch.py

```diff
@@ -147,7 +147,11 @@ def _check_state(self):
             raise ValueError("Batch already committed")
 
     def commit(
-        self, return_commit_stats=False, request_options=None, max_commit_delay=None
+        self,
+        return_commit_stats=False,
+        request_options=None,
+        max_commit_delay=None,
+        exclude_txn_from_change_streams=None,
```

Comment:
Default should be False not None , Right ?

Reply:
Since the type of the field is boolean (primitive) it always gets set to false by default. Eventhough you set None it is like setting false. However, updated all references to False